### PR TITLE
Go 1.15.2

### PR DESCRIPTION
**Pull request by pohly**
This builds on top of PR #751 to avoid code conflicts and updates to Go 1.15.2 without changing how certificates are handled, i.e. the changed default in the Go runtime is reset to what it was before the update.

### CODE REVIEW

File: deploy/kustomize/driver/pmem-csi.yaml [hunk 1]
```diff
@@ -112,6 +112,8 @@ spec:
           value: /tmp/termination-log
         - name: PMEM_CSI_DRIVER_NAME
           value: pmem-csi.intel.com
+        - name: GODEBUG
+          value: x509ignoreCN=0
```
Comment (avalluri) [inline deploy/kustomize/driver/pmem-csi.yaml:116]: The driver deployed via operator also requires this environment change.

Comment (pohly) [reply deploy/kustomize/driver/pmem-csi.yaml:116]: Yes. I thought I'd fire of the PR before calling it a day, but with the early consistency check it didn't get very far... Will update tomorrow...

Comment (pohly) [reply deploy/kustomize/driver/pmem-csi.yaml:116]: Fixed.

**Comment by pohly** [comment]
Updated PR:
- operator itself also needs GODEBUG
- lz4 installed in build container - that was broken in "devel" by the switch to Debian

**Comment by pohly** [comment]
> lz4 installed in build container - that was broken in "devel" by the switch to Debian

That's... hogwash! lz4 is needed on the Jenkins workers. We have an `apt-get install liblz4-tool` in the worker's install script in Jenkins for that. What must have happened in the failed job is that the command failed and the worker came up anyway - a known issue of our CI.

**Comment by pohly** [comment]
@avalluri : this seems to be ready.

**Review by avalluri — APPROVED** [review]

**Pull request merged**